src/components/VaccineProgressBar/VaccineProgressBar.tsx

```diff
@@ -96,8 +98,9 @@ const VaccineProgressBar: React.FC<ProgressBarProps & { width: number }> = ({
 const VaccineProgressBarAutosize: React.FC<ProgressBarProps> = props => {
   return (
     <ProgressBarContainer>
-      <ParentSize>
-        {({ width }) => <VaccineProgressBar {...props} width={width} />}
+      {/* Adding display:flex makes sure the ParentSize container is tight to its contents */}
```

Comment:
im no longer using the autosize version in the vaccinations table, but this fixes a problem i noticed when i was using it